### DISCUSSION TITLE
Is there a way to disable signups?

### DISCUSSION THREAD

**Discussion by voron69-bit** [sysadminsmedia/homebox#214] [Q&A] [answered]
Once we've created our accounts, can we lock this down by disabling sign ups?

**Comment by jared-cormier** [comment, marked as answer]
Yes, edit the Docker Compose file to include "HBOX_OPTIONS_ALLOW_REGISTRATION=false;"  in the environment section

**Reply by voron69-bit** [reply]
Thank You!
